src/modules/settings/components/settings/twitch/EmoteMenu.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Panel from 'rsuite/Panel';
+import {Toggle} from 'rsuite';
+import {registerComponent} from '../../Store.jsx';
+import {CategoryTypes, SettingIds, EmoteMenuTypes} from '../../../../../constants.js';
+import styles from '../../../styles/header.module.css';
+import useStorageState from '../../../../../common/hooks/StorageState.jsx';
+import formatMessage from '../../../../../i18n/index.js';
+
+const SETTING_NAME = formatMessage({defaultMessage: 'Emote Menu'});
+
+function EmoteMenu() {
+  const [value, setValue] = useStorageState(SettingIds.EMOTE_MENU);
+  const toggled = value !== EmoteMenuTypes.NONE;
+  return (
+    <Panel header={SETTING_NAME}>
+      <div className={styles.settingRow}>
+        <p className={styles.settingDescription}>
+          {formatMessage({defaultMessage: 'Enables a more advanced emote menu for chat'})}
+        </p>
+        <Toggle checked={toggled} onChange={(state) => setValue(state ? EmoteMenuTypes.LEGACY : EmoteMenuTypes.NONE)} />
```

Comment:
if we do want to call the old type legacy, perhaps toggling this fresh on sets `ENABLED` and and toggling the following setting off is `LEGACY_ENABLED`